lib/libyui_client/widgets.rb

```diff
@@ -10,5 +10,11 @@ module Widgets
     require 'libyui_client/widgets/radiobutton'
     require 'libyui_client/widgets/table'
     require 'libyui_client/widgets/textbox'
+    require 'libyui_client/widgets/label'
+    require 'libyui_client/widgets/richtext'
+    require 'libyui_client/widgets/dialog'
```

Comment:
We have discussed that with @OleksandrOrlov once, `dialog` is not a widget and strictly speaking, all the widgets belong to dialog. So in case we want to have special handling of it, we can define this structure.

Reply:
we don't check 'a the moment' in the tree of widget if something contains other thing, just plane check, shouldn't be useful to check if it is a popup or other type of container? Maybe not, just asking, so you propose to remove it for now?

Reply:
Adding it as a widget is definitely wrong. So if we want to support it, we need to create proper structure for it. It's definitely will be helpful and handy, so either remove it for now from the PR, or let's add it properly.

Reply:
got it, I will delete it as I guess it is better to wait to work with more modules to see the behavior of those containers and propose an structure.

Reply:
fixed